### PR TITLE
Look for peer nodes at 'dnsseed.argentumcoin.cc' DNS Seeder

### PR DESCRIPTION
**Pull request by dbkeys**

### CODE REVIEW

File: src/chainparams.cpp [hunk 1]
```diff
@@ -165,7 +165,8 @@ class CMainParams : public CChainParams {
         assert(consensus.hashGenesisBlock == uint256S("0x88c667bc63167685e4e4da058fffdfe8e007e5abffd6855de52ad59df7bb0bb2"));
         assert(genesis.hashMerkleRoot == uint256S("0xc7e4af4190d3eb28d4e61261fb1b47e13d7efd6d7279c75ae6a91c2987d43d53"));
 
-        vSeeds.push_back(CDNSSeedData("", ""));
+	// Add DNS Node Tracker/Seeder 'dnsseed.argentumcoin.cc'  ( Jan 23, 2021 )
+        vSeeds.push_back(CDNSSeedData("argentumcoin.cc", "dnsseed.argentumcoin.cc"));
```
Comment (dbkeys) [inline src/chainparams.cpp]: DNS Node Tracker/Seeder for subdomain "dnsseed.argentumcoin.cc" is operational.

Verify operation with 'dig' command:

root~# dig dnsseed.argentumcoin.cc

; <<>> DiG 9.11.3-1ubuntu1.13-Ubuntu <<>> dnsseed.argentumcoin.cc
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 62444
;; flags: qr rd ra; QUERY: 1, ANSWER: 2, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 512
;; QUESTION SECTION:
;dnsseed.argentumcoin.cc.       IN      A

;; ANSWER SECTION:
dnsseed.argentumcoin.cc. 3599   IN      A       85.15.179.171
dnsseed.argentumcoin.cc. 3599   IN      A       95.79.35.133

;; Query time: 619 msec
;; SERVER: 8.8.8.8#53(8.8.8.8)
;; WHEN: Sun Jan 24 01:52:32 UTC 2021
;; MSG SIZE  rcvd: 84

**Review by Cryptcollector — APPROVED** [review]
Just approved these changes to the DNS Seed

**Comment by dbkeys** [comment]
Pull request still open ..

**Comment by dbkeys** [comment]
Outdated

**Pull request closed without merging**